asctb-api/src/index.ts

```diff
@@ -1,255 +1,28 @@
-  const sheetID = req.params.sheetid;
-  const gID = req.params.gid;
-  try {
-    let resp: any;
-
-    if (sheetID === '0' && gID === '0') {
-      resp = { data: PLAYGROUND_CSV };
-    } else {
-      resp = await axios.get(
-        `https://docs.google.com/spreadsheets/d/${sheetID}/export?format=csv&gid=${gID}`
-      );
-    }
-    const data = papa.parse(resp.data).data;
-    const asctbData = await makeASCTBData(data);
-    const graphData = makeGraphData(asctbData);
-
-    return res.send({
-      data: graphData,
-      csv: resp.data,
-      parsed: data,
-    });
-  } catch (err) {
-    console.log(err);
-    return res.status(500).send({
-      msg: 'Please check the table format or the sheet access',
-      code: 500,
-    });
-  }
-});
-
-app.get('/v2/csv', async (req: express.Request, res: express.Response) => {
-  console.log(`${req.protocol}://${req.headers.host}${req.originalUrl}`);
-  // query parameters csvUrl and output
-  const url = req.query.csvUrl as string;
-  const output = req.query.output as 'json' | 'graph' | string;
-
-  try {
-    const response = await axios.get(url);
-
-    const data = papa.parse(response.data, {skipEmptyLines: 'greedy'}).data;
-    const asctbData = await makeASCTBData(data);
-
-    if (output === 'graph') {
-      const graphData = makeGraphData(asctbData);
-      return res.send({
-        data: graphData,
-        csv: response.data,
-        parsed: data,
-      });
-    } else {
-      // The default is returning the json
-      return res.send({
-        data: asctbData,
-        csv: response.data,
-        parsed: data,
-      });
-    }
-
-  } catch (err) {
-    console.log(err);
-    return res.status(500).send({
-      msg: 'Please check the CSV format',
-      code: 500,
-    });
-  }
-});
-
-app.get('/v2/playground', async (req: express.Request, res: express.Response) => {
-  console.log(`${req.protocol}://${req.headers.host}${req.originalUrl}`);
-  try {
-    const parsed = papa.parse(PLAYGROUND_CSV).data;
-    const data = await makeASCTBData(parsed);
-    return res.send({
-      data: data,
-      csv: PLAYGROUND_CSV,
-      parsed: parsed,
-    });
-  } catch (err) {
-    console.log(err);
-    return res.status(500).send({
-      msg: JSON.stringify(err),
-      code: 500,
-    });
-  }
-});
-
-app.post('/v2/playground', async (req: express.Request, res: express.Response) => {
-  const csv = papa.unparse(req.body);
-  try {
-    const data = await makeASCTBData(req.body.data);
-    return res.send({
-      data: data,
-      parsed: req.body,
-      csv: csv,
-    });
-  } catch (err) {
-    console.log(err);
-    return res.status(500).send({
-      msg: JSON.stringify(err),
-      code: 500,
-    });
-  }
-});
-
-app.get('/', (req: express.Request, res: express.Response) => {
-  res.sendFile('views/home.html', { root: path.join(__dirname, '../../') });
-});
-
-app.get('/:sheetid/:gid', async (req: express.Request, res: express.Response) => {
```

Comment:
@bherr2 By the way, I deleted this old v1 route and never moved it to a new file. I didn't think it was being used anywhere but correct me if I'm wrong.

Reply:
Hmm, ok, that's fine. Thanks for letting me know before I merged it.